src/pages/blocks/[blockId]/index.tsx

```diff
@@ -0,0 +1,221 @@
+  text: '',
+  label: ''
+}
+
+export default function BlockDetails ({ block, confirmations, transactions }: InferGetServerSidePropsType<typeof getServerSideProps>): JSX.Element {
+  const leftBlockDetails = [
+    {
+      label: 'Block Reward:',
+      text: 'xxxx DFI'
+    },
+    {
+      label: 'Height:',
+      text: `${block.height}`,
+      testId: 'block-detail-height'
+
+    },
+    {
+      label: 'Transactions:',
+      text: `${block.transactionCount}`,
+      testId: 'block-detail-transactions'
+    },
+    {
+      abel: 'Timestamp:',
+      text: `${format(fromUnixTime(block.medianTime), 'PPpp')}`,
+      testId: 'block-detail-timestamp'
+    },
+    {
+      label: 'Confirmations:',
+      text: `${confirmations}`,
+      testId: 'block-detail-confirmations'
+    },
+    {
+      label: 'Merkle Root:',
+      children:
+  <>
+    <span className='flex-grow break-all' data-testid='block-detail-merkle-root'>{block.merkleroot}</span>
+    <CopyButton text={block.merkleroot} />
+  </>
+    }
+  ]
+
+  const rightBlockDetails = [
+    {
+      label: 'Difficulty:', text: `${block.difficulty}`, testId: 'block-detail-difficulty'
+    },
+    {
+      label: 'Bits:', text: `${block.weight}`
+    },
+    {
+      label: 'Size (bytes):', text: `${block.size}`, testId: 'block-detail-size'
+    },
+    {
+      label: 'Version:', text: `${block.version}`, testId: 'block-detail-version'
+    },
+    {
+      label: 'Next Block:', text: 'next block (placeholder)'
+    },
+    {
+      label: 'Previous Block:', text: 'previous block (placeholder)'
+    }
+
+  ]
+
+  return (
+    <div className='container mx-auto px-4 py-8'>
+      <div className='flex items-center justify-center pb-6'>
+        <div className='bg-pink-50 rounded p-3'>
+          🚧 Work in progress, this is an early iteration of defiscan.live/blocks/[blockHash]. Some features are not available and
+          may not work as expected.
+        </div>
+      </div>
+      <Breadcrumb items={[
+        { path: '/blocks', name: 'Blocks' },
+        { path: `/blocks/${block.height}`, name: `#${block.height}`, canonical: true }
+
+      ]}
+      />
+
+      <h1 className='font-semibold text-2xl'>Block #{block.height}</h1>
+      <div className='flex items-center'><span className='font-semibold'>Hash:&nbsp;</span> <span className='text-primary' data-testid='block-hash'>{block.hash}</span> <CopyButton text={block.hash} /></div>
+      <div className='flex mt-6 gap-x-6'>
+        <div className='flex flex-col w-5/12 flex-grow rounded border divide-solid divide-y-2'>
+          {
+            leftBlockDetails.map(detail => <BlockDetail {...detail} key={detail.label} />)
+          }
+        </div>
+        <div className='flex flex-col w-5/12 flex-grow rounded border divide-solid divide-y-2'>
+          {
+            rightBlockDetails.map(detail => <BlockDetail {...detail} key={detail.label} />)
+          }
+        </div>
+      </div>
+      <div className='my-6'>
```

Comment:
![image](https://user-images.githubusercontent.com/4266087/131492060-870a7acb-3700-4b87-b7b1-7f4674f77be8.png)

Where is this?

Reply:
Added